### PR TITLE
GH-10: add keyboard ui on dashboard screen

### LINKED ISSUE #10: Add UI for keyboard
we need to add UI for the keyboard according to the draft design

### PR DESCRIPTION
**Pull request by alex-waverley**
add UI for the keyboard according to the draft design
Closes #10

### CODE REVIEW

File: frontend/src/components/keyboard/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <SimpleKeyboard
+                layoutName={layoutName}
+                onKeyPress={onKeyPress}
+                theme={'hg-theme-default custom-theme-dark'}
+                baseClass={'simple-keyboard-main'}
+                physicalKeyboardHighlight={true}
+                syncInstanceInputs={true}
+                mergeDisplay={true}
+                layout={{
+                    default: [
```
Comment (sskorol) [inline frontend/src/components/keyboard/index.js:29]: Let's probably extract these layout and display settings from the component into constants or separate keyboard configs.

**Review by sskorol — CHANGES_REQUESTED** [review]
@alex-waverley check the comment, please. And also, add the `Closes #10` part into descritpion.

File: frontend/src/components/keyboard/index.js (hunk 1 above)
Comment (alex-waverley) [reply frontend/src/components/keyboard/index.js]: done

**Review by sskorol — APPROVED** [review]
LGTM

**Pull request merged**